cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java

```diff
@@ -68,6 +68,7 @@
             ProjectDirConstants.SETTINGS_FILE_NAME);
     private static PrintStream outStream = System.err;
     private static EmbeddedExecutor executor = EmbeddedExecutorProvider.getInstance().getExecutor();
+    private static Settings settings;
```

Comment:
Cant we assign the variable here it self?

Reply:
Fixed with fb2abec

Reply:
If its for a first time user then we have to read the Settings.toml constantly until the access token is updated.